rewrite-maven/src/main/java/org/openrewrite/maven/MavenExecutionContextView.java

```diff
@@ -253,6 +270,10 @@ public MavenExecutionContextView setMavenSettings(@Nullable MavenSettings settin
         return effectiveSettings;
     }
 
+    public boolean hasNoDefinedConfiguration() {
+        return getSettings() == null && getLocalRepository().equals(MAVEN_LOCAL_DEFAULT) && getRepositories().isEmpty() && getActiveProfiles().isEmpty() && getMirrors().isEmpty();
+    }
```

Comment:
While I understand adding this here, can we instead inline this in MavenPomDownloader? The reason is this class is parent classloaded, so then it's a more complicated coordinated roll out.